test/Microsoft.AspNet.Mvc.Core.Test/ResponseCacheAttributeTest.cs

```diff
@@ -199,5 +325,22 @@ private IServiceProvider GetServiceProvider(Dictionary<string, CacheProfile> cac
 
             return serviceProvider.Object;
         }
+
+        private ActionExecutingContext GetActionExecutingContext(List<IFilterMetadata> filters = null)
```

Comment:
Why does the `filters` parameter have a default value? Always called with a value.

Any why `List<IFilterMetadata>` instead of the easier-to-use `params IFilterMetadata[]`?